pdata/pmetric/json.go

```diff
@@ -88,20 +88,7 @@ func (d *jsonUnmarshaler) readResourceMetrics(iter *jsoniter.Iterator) *otlpmetr
 	iter.ReadObjectCB(func(iter *jsoniter.Iterator, f string) bool {
 		switch f {
 		case "resource":
-			iter.ReadObjectCB(func(iter *jsoniter.Iterator, f string) bool {
-				switch f {
-				case "attributes":
-					iter.ReadArrayCB(func(iter *jsoniter.Iterator) bool {
-						rs.Resource.Attributes = append(rs.Resource.Attributes, json.ReadAttribute(iter))
-						return true
-					})
-				case "droppedAttributesCount", "dropped_attributes_count":
-					rs.Resource.DroppedAttributesCount = json.ReadUint32(iter)
-				default:
-					iter.Skip()
-				}
-				return true
-			})
+			json.ReadResource(iter, &rs.Resource)
```

Comment:
I remember to see unit tests for this, you should remove them.

Reply:
Sorry, I don't understand what you mean. Which part of the unit test do I need to remove?